pkg/controller/contrailcni/daemonset.go

```diff
@@ -0,0 +1,206 @@
+package contrailcni
+
+import (
+	apps "k8s.io/api/apps/v1"
+	core "k8s.io/api/core/v1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type CniDirs struct {
+	BinariesDirectory string
+	DeploymentType    string
+}
+
+//GetDaemonset returns DaemonSet object for vRouter CNI
+func GetDaemonset(cniDir CniDirs, requestName, instanceType string) *apps.DaemonSet {
+	var trueVal = true
+
+	var podIPEnv = core.EnvVar{
```

Comment:
I don't think you need the pod ip in any of the containers here.

Reply:
Done